### ISSUE TITLE
Multipass instance crashing when starting on Mac M1

### ISSUE THREAD

**Issue by derekmizak** [canonical/multipass#2601]
Multipass instance crashing when starting:

`15:42 @D3~>multipass start wizard -vvvv
[2022-05-30T15:43:11.914] [debug] [wizard] process working dir ''               
[2022-05-30T15:43:11.914] [info] [wizard] process program 'qemu-system-aarch64' 
[2022-05-30T15:43:11.914] [info] [wizard] process arguments '-machine, virt,highmem=off, -accel, hvf, -drive, file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on, -cpu, cortex-a72, -nic, vmnet-shared,model=virtio-net-pci,mac=52:54:00:57:36:88, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/wizard/ubuntu-20.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 2, -m, 8192M, -qmp, stdio, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/root/Library/Application Support/multipassd/qemu/vault/instances/wizard/cloud-init-config.iso'
[2022-05-30T15:43:11.924] [debug] [qemu-system-aarch64] [3899] started: qemu-system-aarch64 -machine virt,highmem=off -nographic -dump-vmstate /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/multipassd.FXrWHv
[2022-05-30T15:43:11.982] [info] [wizard] process state changed to Starting     
[2022-05-30T15:43:11.984] [info] [wizard] process state changed to Running      
[2022-05-30T15:43:11.984] [debug] [qemu-system-aarch64] [3900] started: qemu-system-aarch64 -machine virt,highmem=off -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu cortex-a72 -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:57:36:88 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/wizard/ubuntu-20.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 8192M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/wizard/cloud-init-config.iso
[2022-05-30T15:43:11.985] [info] [wizard] process started                       
start failed: The following errors occurred:                                    
wizard: timed out waiting for response
15:48 @D3~>
`

After that instance is in the Unknown stage and I am unable to stop it. 

**Additional info**
 - `OS: [macOS 12.4]`
 - `MacBook Pro M1`
- `multipass   1.9.0+mac`
- `multipassd  1.9.0+mac`

**Comment by luis4a0** [comment]
Hi @derekmizak, MacOS changes the firewall rules when upgrading to 12.4. For this reason, it is possible the instances become unreachable. Please refer to #2387 for insights on how to fix this.

TL;DR: try with rebooting, disabling/reenabling firewall and/or internet sharing.

Duplicate of #2387.
